### ISSUE TITLE
V12: Adding new parameter value

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/MoBi#1676]
When I manually add a new parameter value: it's always added with the value 0 and dimension "Dimensionless"
(the warning below comes)
<details><summary>Select parameter</summary>

![grafik](https://github.com/user-attachments/assets/20a8e419-a7a7-4afd-94cf-665aa1b0f157)
</details> 

Any reason why the value and the dimension are not inherited from the selected parameter source?
![{0DFCED52-E2F4-45A3-ACCE-26B9CBED33DB}](https://github.com/user-attachments/assets/dbe9ad82-7463-4175-abce-3060f9d45c71)

**Comment by rwmcintosh** [comment]
Yeah, we could add that. As originally conceived the dialog here creates a string representation of the selected parameter path. Then, you could edit the paths by typing, so at this stage, the paths have nothing to do with the original selections.

But we could make it such that the (possibly modified) paths are then resolved when the user presses the OK button and if a parameter can be found, we could set the dimension and initial value.

**Comment by PavelBal** [comment]
I'd say "nice to have, but can wait for vNext".
